ios/Classes/MethodCallHandler.swift

```diff
@@ -275,6 +275,9 @@ extension VideoType {
 
 extension Dictionary where Key == String {
     func toVideoOptions() -> VideoOptions {
-        return VideoOptions(videoId: self["videoId"] as! String, videoType: (self["videoType"] as! String).toVideoType())
+        guard let token = self["token"] as? String else {
+            return VideoOptions(videoId: self["videoId"] as! String, videoType: (self["videoType"] as! String).toVideoType())
+        }
+        return VideoOptions(videoId: self["videoId"] as! String, videoType: (self["videoType"] as! String).toVideoType(), token: token)
     }
 }
```

Comment:
Ok, this is the implementation for the `setVideoOptions` what about the `getVideoOptions`?